tasks/save-collectors-results/save-collectors-results.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: save-collectors-results
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to update the passed CR status with the
+    collectors results.
+  params:
+    - name: resourceType
+      description: The type of resource that is being patched
+      type: string
+      default: release
+    - name: statusKey
+      description: The top level key to overwrite in the resource status
+      type: string
+      default: collectors
+    - name: resource
+      description: The namespaced name of the resource to be patched
+      type: string
+    - name: resultsDirPath
+      description: |
+        The relative path in the workspace where the collectors results
+        are saved to
+      type: string
+  workspaces:
+    - name: data
+      description: Workspace where the results directory is stored
+  steps:
+    - name: save-collectors-results
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      script: |
+        #!/usr/bin/env bash
+        set -ex
+
+        RESULTS_JSON="{}"
+        RESULTS_DIR="$(workspaces.data.path)/$(params.resultsDirPath)"
+        for resultsFile in $([ -d "$RESULTS_DIR" ] && find "$RESULTS_DIR" -type f); do
+            if ! jq . >/dev/null 2>&1 "${resultsFile}" ; then
+                echo "Ignoring not JSON file: ${resultsFile}."
```

Comment:
If you want to ignore such file, then this should also run `continue`. Without it, we would try to process it even if it's invalid json, but has the right filename.

Reply:
Absolutely right.